joints: ['arm_shoulder_pan_joint', 'arm_shoulder_lift_joint', 'arm_wrist_flex_joint', 'gripper_joint']
arm_shoulder_pan_joint:
  controller:
    package: dynamixel_controllers
    module: joint_torque_controller
    type: JointTorqueController
  joint_name: arm_shoulder_pan_joint
  joint_speed: 1.5
  motor:
    id: 8
    init: 512
    min: 0
    max: 1023
arm_shoulder_lift_joint:
  controller:
    package: dynamixel_controllers
    module: joint_torque_controller
    type: JointTorqueController
  joint_name: arm_shoulder_lift_joint
  joint_speed: 1.0
  motor:
    id: 4
    init: 512
    min: 200
    max: 800
arm_wrist_flex_joint:
  controller:
    package: dynamixel_controllers
    module: joint_torque_controller
    type: JointTorqueController
  joint_name: arm_wrist_flex_joint
  joint_speed: 1.0
  motor:
    id: 5
    init: 512
    min: 200
    max: 800
gripper_joint:
  controller:
    package: dynamixel_controllers
    module: joint_torque_controller
    type: JointTorqueController
  joint_name: gripper_joint
  joint_speed: 1.5
  motor:
    id: 7
    init: 512
    min: 0
    max: 900
arm_controller:
  controller:
    package: dynamixel_controllers
    module: joint_trajectory_action_controller
    type: JointTrajectoryActionController
  joint_trajectory_action_node: 
    min_velocity: 0.1
    constraints:
         goal_time: 0.1
